### ISSUE TITLE
use case: data fingerprinting/auditing

### ISSUE THREAD

**Issue by dberenbaum** [iterative/dvc.org#4654]
### Report

Consider how we can better explain how DVC can be useful for data fingerprinting and auditing, especially for highly regulated industries like healthcare and banking.

cc @efiop @shcheklein

**Comment by shcheklein** [comment]
cc @jurv11  - I think it should be on the dvc.org, but also a blog post, + some updates to Iterative.ai

**Comment by shcheklein** [comment]
@dberenbaum @jurv11 any action items here?

**Comment by dberenbaum** [comment]
@shcheklein Any thoughts on whether this should be a priority?

**Comment by shcheklein** [comment]
Yep, at least seems to be part of the effort that we've been discussing - business / high level value of the ecosystem.
